code/modules/mob/living/simple_animal/abnormality/teth/blood_bath.dm

```diff
@@ -16,7 +16,7 @@
 	work_damage_type = WHITE_DAMAGE
 	max_boxes = 14
 	ego_list = list(
-		/datum/ego_datum/weapon/bloodbath,
+	//	/datum/ego_datum/weapon/wrist,
```

Comment:
Feel like this should've been atomized?

Reply:
It's a fix, weapon is for some reason just not here.